### DISCUSSION TITLE
Navigasjon

### DISCUSSION THREAD

**Discussion by mrosvik** [digdir/designsystemet#1510] [Patterns for public services]
## Relevant

- https://github.com/digdir/innsiktsbibliotek/issues/25
- https://github.com/digdir/innsiktsbibliotek/issues/24
- https://github.com/digdir/innsiktsbibliotek/issues/21
- https://aksel.nav.no/god-praksis/artikler/navigasjon-i-skjemaer?tema=skjemaer

-----

**Comment by solhell** [comment]
WCAG stiller krav til minst to metoder for å finne frem til innhold.
Noen aktuelle metoder er meny, innholdsnavigering, søk, alfabetisk liste, emnekart og nettstedskart.
Mitt innspill er at det bør være to metoder som dekker absolutt alt innholdet, utenom det som krever at man går gjennom en spesifikk arbeidsflyt.

Det jeg ofte ser er at menyen dekker innholdet ned til et visst nivå, men deretter endrer navigeringsprinsippet seg til at man må navigere gjennom innholdet. For brukere med kognitive utfordringer eller som er digitalt uvant kan dette skiftet i navigasjonsprisnipp være utfordrende - man må lære seg når man kan bruke menyen og når man må navigere gjennom innholdet. Og gjennom en brukersesjon kan dette endre seg ganske mye frem og tilbake, avhengig av hvilken oppgave brukeren ønsker å løse, i kombinasjon med hvor lett hen har for å finne frem til riktig sted. Og ikke minst mellom brukersesjoner der brukeren kommer med ulike oppgaver for hver gang.

**Comment by mrosvik** [comment]
Kanskje aktuelt for dette mønsteret: **Navigasjon i dokumenter presentert på nettsider**

Eksempel fra regjeringen: 
<img width="1091" height="900" alt="image" src="https://github.com/user-attachments/assets/703ed04b-abf9-4bc2-b4e1-45e6b25d1c3e" />
